src/data/icons/svg/HBAR.svg

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   id="Layer_1"
```

Comment:
we need cleaner .svg that don't have "id" and "defs" in it, because it wouldn't work when converted to components (like React Native components)

Reply:
@gre we decided to have it designed internally
(cc @rbeyon-ledger )